domains/zeus.json

```
@@ -0,0 +1,13 @@
+{
+    "description": "Portfolio",
+    "repo": "https://github.com/t1devzeus/register",
+    "owner": {
+        "username": "t1devzeus",
+        "discord": "944959943310737408"
+    },
```

you need to add your email or add it as an empty string
```suggestion
    "owner": {
        "username": "t1devzeus",
        "email": "",
        "discord": "944959943310737408"
    },
```

---

Done